src/CDSAPI.jl

```diff
@@ -3,6 +3,72 @@ module CDSAPI
 using HTTP
 using JSON
 
+if VERSION >= v"1.11"
+    using Base.ScopedValues
+elseif VERSION >= v"1.8"
+    import Pkg
+    Pkg.activate(".")
+    Pkg.add("ScopedValues")
+    using ScopedValues
+end
```

Comment:
The idea is to run these commands in the source code to modify the Project.toml of the package. There is no problem in adding ScopedValues.jl as a dependency.

Reply:
Ok, so then just remove the `Pkg.add("ScopedValues")`?